modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
That's not why am suggesting to put it on the button

If it helps, think of the icon as a "visual label" for sighted people, telling them "that is what this button does"